source/includes/api/_query.md

```diff
@@ -375,6 +417,29 @@ curl -i -X GET 'https://api.resourcewatch.org/v1/download/9be3bf63-97fc-4bb0-b91
 
 When downloading the query results for a document based dataset using either GET or POST `/download/<dataset id or slug>` request, the `FROM` clause is required but ignored, meaning you don't have to provide the dataset's `tableName` as you normally would. The example on the side illustrates this.
 
+## Deleting data from a dataset
```

Comment:
I'm not sure, but I suspect this process is asynchronous. Please test it, and if so, document its details, so users know that the dataset will go to "pending" state and they have to wait before using it.

Reply:
Done :)